include/sundials/sundials_base.hpp

```diff
@@ -119,32 +120,48 @@ class ClassView : public sundials::ConvertibleTo<T>
 public:
   ClassView() : object_(nullptr) {}
 
-  ClassView(T&& object) : object_(std::make_unique<T>(object)) {}
+  ClassView(T& object) : object_(object) {}
 
-  ClassView(const ClassView&)  = delete;
-  ClassView(ClassView&& other) = default;
+  ClassView(T&& object) : object_(object) {}
+
+  ClassView(const ClassView&) = delete;
+
+  ClassView(ClassView&& other) noexcept
+    : object_(std::exchange(other.object_, nullptr))
+  {}
 
   ClassView& operator=(const ClassView&) = delete;
-  ClassView& operator=(ClassView&& rhs)  = default;
+
+  ClassView& operator=(ClassView&& rhs) noexcept
+  {
+    this->object_ = std::exchange(rhs.object_, nullptr);
+    return *this;
+  };
 
   ~ClassView()
   {
-    if (object_) { Deleter{}(this->Convert()); }
+    if (object_) { Deleter{}(this->get()); }
   };
 
   // Override ConvertibleTo functions
-  T Convert() override { return *object_.get(); }
+  T get() override { return object_; }
 
-  T Convert() const override { return *object_.get(); }
+  T get() const override { return object_; }
```

Comment:
I'm guessing that this is the cause for the member function changes elsewhere in our C++ headers.  I don't fully follow the need for this, but I'm ready to take your word on it.

Reply:
Explained elsewhere, but basically this function will be typed many times by Python users so a shorter name is needed, also we decided on this many months ago.